src/common/runtime/src/runtime.rs

```diff
@@ -24,7 +24,7 @@ use tokio::sync::oneshot;
 pub use tokio::task::{JoinError, JoinHandle};
 
 use crate::error::*;
-use crate::metric::*;
+use crate::metrics as constant_metrics;
```

Comment:
nitpick: Could we avoid the `as`?
```suggestion
use crate::metrics;
```

Reply:
I could avoid the `as` by change `metrics.rs` to other.